library/VTKExtensions/Rendering/vtkF3DDropZoneActor.cxx

```diff
@@ -0,0 +1,167 @@
+#include "vtkF3DDropZoneActor.h"
+
+#include <vtkObjectFactory.h>
+#include <vtkPolyData.h>
+#include <vtkPolyDataMapper2D.h>
+#include <vtkTextMapper.h>
+#include <vtkTextProperty.h>
+#include <vtkViewport.h>
+
+constexpr int tickWidth = 3;
+constexpr int tickLength = 10;
```

Comment:
only used in `BuildBorderGeometry`, please move the declaration inside, there's no reason to declare it globally.